lib/jinja2/ext.py

```diff
@@ -596,7 +596,8 @@ def getbool(options, key, default=False):
         getbool(options, 'keep_trailing_newline', KEEP_TRAILING_NEWLINE),
         frozenset(extensions),
         cache_size=0,
-        auto_reload=False
+        auto_reload=False,
+        autoescape=True
```

Comment:
Can you explain what `autoescape` does? (and presumably there's no reason to think it will break our use cases?)

Reply:
(OK looked it up, seems sensible).